build.sbt

```diff
@@ -87,7 +87,11 @@ lazy val rocketchip = dependOnChisel(project in file("."))
       mappings in (Compile, packageBin) ++= (mappings in (`rocket-macros`, Compile, packageBin)).value,
       mappings in (Compile, packageSrc) ++= (mappings in (`rocket-macros`, Compile, packageSrc)).value,
       exportJars := true,
-      Test / unmanagedBase := baseDirectory.value / "test_lib"
+      Test / unmanagedBase := baseDirectory.value / "test_lib",
+      // Settings for scalafix
+      semanticdbEnabled := true,
+      semanticdbVersion := scalafixSemanticdb.revision,
+      scalacOptions += "-Ywarn-unused-import"
```

Comment:
@jackkoenig, let me know if this is the best way to add in the configuration for Scalafix. I don't even know what SemanticDB is, but it said I needed to enable it by adding these lines to the project. `-Ywarn-unused-import` is apparently required for the `RemoveUnused` rule to function correctly.